pkg/project/list.go

```diff
@@ -0,0 +1,43 @@
+package project
```

Comment:
Since `installer` package now needs to refer to this project list, I'm moving this to the `project` though filename may be varied from other PE's opinion.

Reply:
Nice. Good to move it and the name makes sense to me.

Reply:
The name makes sense I agree but we use it for a different purpose. Till now it served information about the project itself (version, source, etc.) and not about its configuration.

I'd propose to move it to `pkg/setttings` or something similar, but not because the name is wrong - I think it may be confusing.

Reply:
WDYT `pkg/configuration/`?

Reply:
going with `/pkg/configuration`

Reply:
`/pkg/configuration` makes sense to me. 👍